### PR TITLE
Rename PriceSchema to AmountSchema

### PR DESCRIPTION
**Pull request by nachoiacovino**
So we get a correct error message if they submit an invalid quantity

### CODE REVIEW

**Review by jnsdls — APPROVED** [review]
this will work fine but I dislike the duplication just for a different error message

File: src/schema/shared.ts [hunk 1]
```diff
@@ -84,6 +84,13 @@ export const RawDateSchema = z.date().transform((i) => {
   return BigNumber.from(Math.floor(i.getTime() / 1000));
 });
 
+export const AirdropQuantitySchema = z
```
Comment (joaquim-verges) [inline src/schema/shared.ts:87]: I think we already have an alias for Price that is "Amount" - can you use that instead?

Comment (nachoiacovino) [reply src/schema/shared.ts:87]: not sure what you mean here, I added this because I realized the error message for airdrop when invalid quantity was wrong

Comment (joaquim-verges) [reply src/schema/shared.ts:87]: lets just call this `AmountSchema` and use it for the `Amount` type

**Review by joaquim-verges — APPROVED** [review]

File: src/common/currency.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ export async function normalizePriceValue(
   currencyAddress: string,
 ) {
   const metadata = await fetchCurrencyMetadata(provider, currencyAddress);
-  return utils.parseUnits(PriceSchema.parse(inputPrice), metadata.decimals);
+  return utils.parseUnits(AmountSchema.parse(inputPrice), metadata.decimals);
```
Comment (joaquim-verges) [inline src/common/currency.ts:35]: this was PriceSchema - why is it changed?

File: src/types/currency.ts [hunk 3]
```diff
@@ -39,10 +39,10 @@ export type TokenHolderBalance = { holder: string; balance: CurrencyValue };
  * Represents a currency price already formatted. ie. "1" for 1 ether.
  * @public
  */
-export type Price = z.input<typeof PriceSchema>;
+export type Price = z.input<typeof AmountSchema>;
```
Comment (joaquim-verges) [inline src/types/currency.ts:42]: why did you change this one? this should not be changed

**Review by joaquim-verges — CHANGES_REQUESTED** [review]

**Review by joaquim-verges — APPROVED** [review]
talked offline
